node.gyp

```diff
@@ -167,9 +167,8 @@
         'src/handle_wrap.cc',
         'src/js_stream.cc',
         'src/node.cc',
-        'src/node_api.cc',
-        'src/node_api.h',
-        'src/node_api_types.h',
+#        'src/node_api.cc',
+        'src/node_api_jsrt.cc',
```

Comment:
Can we just add a "conditions" block which checked the engine type and then removes node_api.cc and adds in node_api_jsrt.cc?